modules/Kconfig.mcuboot

```diff
@@ -154,30 +156,79 @@ config MCUBOOT_BOOTLOADER_MODE_SINGLE_APP
 
 config MCUBOOT_BOOTLOADER_MODE_SWAP_WITHOUT_SCRATCH
 	bool "MCUboot has been configured for swap without scratch operation"
+	select MCUBOOT_BOOTLOADER_MODE_HAS_NO_DOWNGRADE
 	help
 	  MCUboot expects slot0_partition and slot1_partition to be present
 	  in DT and application will boot from slot0_partition.
+	  MCUBOOT_BOOTLOADER_NO_DOWNGRADE should also be selected
+	  if MCUboot has been built with MCUBOOT_DOWNGRADE_PREVENTION.
```

Comment:
> MCUBOOT_BOOTLOADER_NO_DOWNGRADE should also be selected if MCUboot has been built with MCUBOOT_DOWNGRADE_PREVENTION.

What is the motivation for a comment about `MCUBOOT_DOWNGRADE_PREVENTION.` being in the `MCUBOOT_BOOTLOADER_MODE_SWAP_WITHOUT_SCRATCH` help-text?

I don't see it as a problem or anything, but it makes me wonder if maybe there is a copy and paste error that explains it?

Reply:
Downgrade prevention is optional feature for swap algorithms that will prevent booting new image with lower version. 
DirectXIP (not XIP) always has downgrade prevention present.